cmd/bnscli/clitests/send_tokens.test

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+keyfile=`tempfile`
+echo 00wZcK6QrPNAXy2Z3KyhbQx9s3n0vq/P32Z7nWnONQ0n9ftEBQnfp57Ig6BRC8mpYUw9RBiIgfDF5AKJi0vzyQ== | base64 -d > $keyfile
```

Comment:
It would be great to document where you created this in order to make the test case.
If it was hard, maybe we can add a `bnscli keygen` command?
Or course, you will use this hard-coded key here, but we can point people to use `keygen` once to set up new keys for eg. genesis file

Reply:
I will write comment while it is hardcoded and use it in a place where it is actually needed (the signature test case).